packages/mutation-testing-elements/src/components/mutation-test-report-totals.ts

```diff
@@ -143,7 +143,12 @@ export class MutationTestReportTotalsComponent extends LitElement {
   }
 
   private renderRow(name: string, row: MetricsResult, path: string | undefined) {
-    const mutationScoreRounded = row.metrics.mutationScore.toFixed(2);
+    let mutationScore = 0;
+    let mutationScoreRounded = null;
```

Comment:
Please create a helper function to calculate `mutationScoreRounded`, instead of resorting to `let`.